charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -112,6 +112,7 @@ fullnameOverride:
 controller:
   additionalArgs: []
   sdkDebugLog: false
+  loggingFormat: text
```

Comment:
Not now, but we should consider making JSON the default sometime in the future.

Reply:
Looking at the [proposal](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1602-structured-logging/README.md), they seem to have text as default. Do we want to have this changed prematurely, considering that this will render the divergence between the default formats for core k8s and CSI driver's logs, which may not be a good customer experience?

My feeling is that we should keep the parity with k8s on that. When they change the default setting, it will make sense for us to do the same.